app/config/routing.yml

```diff
@@ -1,6 +1,9 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
+fos_oauth_server_token:
```

Comment:
We can always adjust a [security.yml file](https://github.com/Sylius/Sylius/blob/master/app/config/security.yml). I bet it would be enough to add an `api` prefix to [this pattern](https://github.com/Sylius/Sylius/blob/master/app/config/security.yml#L60)